roboto/api_types.py

```diff
@@ -429,6 +624,47 @@ class Update:
     pre_checkout_query: Optional[PreCheckoutQuery] = None
 
 
+@dataclass(frozen=True)
+class BotCommand:
+    """This object represents a bot command."""
+
+    command: str
+    description: str
+
+
+@dataclass(frozen=True)
+class ResponseParameters:
+    """Contains information about why a request was unsuccessful."""
+
+    migrate_to_chat_id: Optional[int]
+    retry_after: Optional[int]
+
+
+@dataclass(frozen=True)
```

Comment:
It doesn't make much sense that this is empty. I checked the documentation and am confused.